Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/base/BasePresenter.java

```diff
@@ -47,39 +43,39 @@ public void checkViewAttached() {
         }
     }
 
-    /**
-     * Encapsulate the result of an rx Observable.
-     * This model is meant to be used by the children presenters to easily keep a reference
-     * to the latest loaded result so that it can be easily emitted again when on configuration
-     * changes.
-     */
-    protected static class DataResult<T> {
-
-        private T mData;
-        private Throwable mError;
-
-        public DataResult(T data) {
-            mData = data;
-        }
-
-        public DataResult(Throwable error) {
-            mError = error;
-        }
-
-        public Single<T> toSingle() {
-            if (mError != null) {
-                return Single.error(mError);
-            }
-            return Single.just(mData);
-        }
-
-        public Observable<T> toObservable() {
-            if (mError != null) {
-                return Observable.error(mError);
-            }
-            return Observable.just(mData);
-        }
-    }
+//    /**
+//     * Encapsulate the result of an rx Observable.
+//     * This model is meant to be used by the children presenters to easily keep a reference
+//     * to the latest loaded result so that it can be easily emitted again when on configuration
+//     * changes.
+//     */
+//    protected static class DataResult<T> {
+//
+//        private T mData;
+//        private Throwable mError;
+//
+//        public DataResult(T data) {
+//            mData = data;
+//        }
+//
+//        public DataResult(Throwable error) {
+//            mError = error;
+//        }
+//
+//        public Single<T> toSingle() {
+//            if (mError != null) {
+//                return Single.error(mError);
+//            }
+//            return Single.just(mData);
+//        }
+//
+//        public Observable<T> toObservable() {
+//            if (mError != null) {
+//                return Observable.error(mError);
+//            }
+//            return Observable.just(mData);
+//        }
+//    }
```

Comment:
This code can essentially be removed, because it is not being used anywhere in the app. I was not sure about removing it, so I commented it out. Let me know if it is ok to remove this.

Reply:
do we want to use some basic mvp for architecture on this app? it'll improve readability and help make testing easier but if people think it's unnecessary we can just do classic android.

Reply:
@kimj It's good to remove crufty, unused code for a clean 2018 starting point. If you want to use MVP, great! Get it into a branch and pull request as part of a completed feature. Everyone volunteering on the team can write features the way they know how. There's no time to waste.

Reply:
It's better to remove than to have unused code in comments getting in the way.